Dockerfile.sriov-network-config-daemon

```diff
@@ -5,7 +5,7 @@ RUN make _build-sriov-network-config-daemon BIN_PATH=build/_output/cmd
 
 FROM quay.io/centos/centos:stream9
 ARG MSTFLINT=mstflint
-RUN ARCH_DEP_PKGS=$(if [ "$(uname -m)" != "s390x" ]; then echo -n ${MSTFLINT} ; fi) && yum -y install pciutils hwdata $ARCH_DEP_PKGS && yum clean all
+RUN ARCH_DEP_PKGS=$(if [ "$(uname -m)" != "s390x" ]; then echo -n ${MSTFLINT} ; fi) && yum -y install hwdata $ARCH_DEP_PKGS && yum clean all
```

Comment:
`pciutils` dependency has been introduced by 

- #781 

I think having this dep in `Dockerfile.sriov-network-config-daemon.rhel7` (downstream image) is ok. 
@SalDaniele @wizhaoredhat any thought on this?

Reply:
Yes, downstream requires this dependency for scripts like https://github.com/openshift/sriov-network-operator/blob/master/bindata/scripts/bf2-switch-mode.sh, as it is a dependency of mstfwreset().

Reply:
Thanks for confirming that

Reply:
Yes we should have this dependency in our downstream dockerfile.